MekHQ/src/mekhq/campaign/mission/atb/AtBScenarioModifier.java

```diff
@@ -48,8 +49,22 @@
 import mekhq.campaign.mission.ScenarioObjective;
 
 @XmlRootElement(name="AtBScenarioModifier")
-public class AtBScenarioModifier {
-
+public class AtBScenarioModifier implements Cloneable {
+
+    public static final String SCENARIO_MODIFIER_ALLIED_GROUND_UNITS = "PrimaryAlliesGround.xml";
+    public static final String SCENARIO_MODIFIER_ALLIED_AIR_UNITS = "PrimaryAlliesAir.xml";
+    public static final String SCENARIO_MODIFIER_LIAISON_GROUND = "LiaisonGround.xml";
+    public static final String SCENARIO_MODIFIER_HOUSE_CO_GROUND = "HouseOfficerGround.xml";
+    public static final String SCENARIO_MODIFIER_INTEGRATED_UNITS_GROUND = "IntegratedAlliesGround.xml";
+    public static final String SCENARIO_MODIFIER_LIAISON_AIR = "LiaisonAir.xml";
+    public static final String SCENARIO_MODIFIER_HOUSE_CO_AIR = "HouseOfficerAir.xml";
+    public static final String SCENARIO_MODIFIER_INTEGRATED_UNITS_AIR = "IntegratedAlliesAir.xml";
+    public static final String SCENARIO_MODIFIER_TRAINEES_AIR = "AlliedTraineesAir.xml";
+    public static final String SCENARIO_MODIFIER_TRAINEES_GROUND = "AlliedTraineesGround.xml";
+    public static final String SCENARIO_MODIFIER_ALLIED_GROUND_SUPPORT = "AlliedGroundSupportImmediate.xml";
+    public static final String SCENARIO_MODIFIER_ALLIED_AIR_SUPPORT = "AlliedAirSupportImmediate.xml";
+    public static final String SCENARIO_MODIFIER_ALLIED_ARTY_SUPPORT = "AlliedArtillerySupportImmediate.xml";
```

Comment:
Can we move these to MekHqConstants too, so we have a single location with all filenames and paths (eventually speaking)